### ISSUE TITLE
DBus installation issue related to systemd

### ISSUE THREAD

**Issue by guest1604** [easybuilders/easybuild-easyconfigs#7435]
On CentOS 7, DBus installation fails with this error :
```
$ eb DBus-1.13.6-GCCcore-7.3.0.eb --robot
```
```
 /bin/install -c -m 644 dbus.service dbus.socket '/usr/lib/systemd/system'
/bin/install: cannot remove '/usr/lib/systemd/system/dbus.service': Permission denied
/bin/install: cannot remove '/usr/lib/systemd/system/dbus.socket': Permission denied
```
Adding this option in the easyconfig file solved this issue : `configopts = '--disable-systemd'`

Do you think this is the good solution ?

**Comment by akesandgren** [comment]
To solve that specific problem yes. But DBus belongs among the OS distro packages and should most likely not be built in EB in my opinion.

**Comment by boegel** [comment]
@guest1604 already done recently in #7373

**Comment by jose-d** [comment]
@akesandgren :thinking:

**Comment by boegel** [comment]
Closing this since fixed in #7373
